src/verinfast/cloud/aws/costs.py

```diff
@@ -35,7 +35,7 @@ def _get_costs_and_usage(profile: str, aws_output_file: str):
             raise Exception("Error getting aws cli data.")
 
         text = results.stdout.decode()
-        if text is None or isinstance(text, str) is False:
+        if text is None or text == '' or not isinstance(text, str):
```

Comment:
`if not text or text is not str:`

Reply:
Fixed. And I did some print statements (removed then) to test that "not string" really does check for empty string.

Reply:
![image](https://github.com/user-attachments/assets/92296cc4-3233-4891-8850-037510971975)
Same, bro. same